languages/de-DE/Import.multids

```diff
@@ -22,7 +23,11 @@ Upgrader/Plugins/Suppressed/Incompatible: Unterdrückte, inkompatible oder veral
 Upgrader/Plugins/Suppressed/Version: Einige "plugins" weden unterdrückt! Importierte plugins: <<incoming>> sind älter als existierende: <<existing>>.
 Upgrader/Plugins/Upgraded:  Aktualisieren der plugins von: <<incoming>> nach: <<upgraded>>.
 Upgrader/State/Suppressed: Unterdrückte temporäre Status Tiddler.
+Upgrader/System/Disabled: Deaktivierter "System Tiddler".
```

Comment:
```suggestion
Upgrader/System/Disabled: Deaktivierter "System-Tiddler".
```
Would be great if you could fix the incorrect spaces in the line above  and the ones below this too while you're at it, to keep it consistent! ("System-Tiddler", "Core-Modul-Tiddler", "Core-Tiddler")